bip-0159.mediawiki

```diff
@@ -34,6 +34,8 @@ This BIP proposes a new service bit
 | NODE_NETWORK_LIMITED || bit 10 (0x400) || If signaled, the peer <I>MUST</I> be capable of serving at least the last 288 blocks (~2 days).
 |}
 
+Pruned/limited peers <I>MUST</I> have <code>NODE_NETWORK_LIMITED</code> set and any service bit that signals serving the complete block chain (e.g. <code>NODE_NETWORK</code>) <I>MUST NOT</I> be set. Signaling serving all historical blocks overrides <code>NODE_NETWORK_LIMITED</code>.
```

Comment:
Actually, seeing line 34, defining that `NODE_NETWORK_LIMITED` means that a node is capable of serving _at least_ the last 288 blocks makes me think that it would be correct for nodes with the entire blockchain to indicate both `NODE_NETWORK` and `NODE_NETWORK_LIMITED`.

I don’t understand the motivation for the first half-sentence. A pruned node should be permitted to not signal `NODE_NETWORK_LIMITED`. As far as I understand you want to clarify that a pruned node should not be advertising `NODE_NETWORK`?

```suggestion
Pruned/limited peers <I>MUST NOT</I> set a service bit that signals serving the complete block chain (e.g. <code>NODE_NETWORK</code>.
```

Reply:
Done, nice simplification.